.github/buildomat/jobs/test.sh

```diff
@@ -2,12 +2,11 @@
 #:
 #: name = "test"
 #: variety = "basic"
-#: target = "lab-2.0-opte-0.23"
+#: target = "helios-2.0"
```

Comment:
The only reason we were running this on a lab machine before was to get a sufficiently up to date helios image. Now that the helios-2 image series is available, lab machine use is no longer needed.